### PR TITLE
Add required scopes to endpoints

### PR DESCRIPTION
**Pull request by nunofgs**
R=@fixe

### CODE REVIEW

File: _cards.md [hunk 1]
```diff
@@ -133,6 +137,8 @@ curl https://api.uphold.com/v0/me/cards \
 
 `POST https://api.uphold.com/v0/me/cards`
 
+<aside class="notice">Requires the `accounts:write` scope for Uphold Connect applications.</aside>
```
Comment (fixe) [inline _cards.md]: cards

File: _transactions.md [hunk 2]
```diff
@@ -148,6 +154,8 @@ Triggers a reminder for a transaction that hasn't been redeemed yet.
 
 `POST https://api.uphold.com/v0/me/cards/:card/transactions/:id/resend`
 
+<aside class="notice">Requires the `transactions:send` or `transactions:write` scope for Uphold Connect applications.</aside>
```
Comment (fixe) [inline _transactions.md]: This should be updated.

Comment (rplopes) [inline _transactions.md]: Updated in https://github.com/uphold/docs/pull/95.

File: _transactions.md [hunk 3]
```diff
@@ -92,6 +92,8 @@ Upon preparing a transaction, a [Transaction Object](#transaction-object) will b
 
 `POST https://api.uphold.com/v0/me/cards/:card/transactions`
 
+<aside class="notice">Requires the `transactions:deposit`, `transactions:send`, `transactions:transfer`, `transactions:withdraw` or `transactions:write` scope for Uphold Connect applications.</aside>
```
Comment (fixe) [inline _transactions.md]: Remove.

File: _transactions.md [hunk 4]
```diff
@@ -106,6 +108,8 @@ Once a transaction has been created and a quote secured, commit the transaction
 
 `POST https://api.uphold.com/v0/me/cards/:card/transactions/:id/commit`
 
+<aside class="notice">Requires the `transactions:deposit`, `transactions:send`, `transactions:transfer`, `transactions:withdraw` or `transactions:write` scope for Uphold Connect applications.</aside>
```
Comment (fixe) [inline _transactions.md]: Remove.

File: _transactions.md [hunk 5]
```diff
@@ -126,6 +130,8 @@ Cancels a transaction that has not yet been redeemed.
 
 `POST https://api.uphold.com/v0/me/cards/:card/transactions/:id/cancel`
 
+<aside class="notice">Requires the `transactions:send` or `transactions:write` scope for Uphold Connect applications.</aside>
```
Comment (fixe) [inline _transactions.md]: Remove.

**Pull request merged**